test/test_quantization.py

```diff
@@ -765,5 +767,26 @@ def test_observer_scriptable(self):
         loaded = torch.jit.load(buf)
         self.assertEqual(obs.calculate_qparams(), loaded.calculate_qparams())
 
+    @given(qdtype=st.sampled_from((torch.qint8, torch.quint8)),
+           qscheme=st.sampled_from((torch.per_tensor_affine, torch.per_tensor_symmetric)))
```

Comment:
The new test code also checks for reduce_range, please rebase and merge into that